RamilyaNigmatullina/pdp-2
# [#159385495] Add filter by radius

# Summary

User should be able to filter hotels by radius 

https://www.pivotaltracker.com/story/show/159385495

# Test plan

List of steps to manually test introduced functionality:

* Go to Hotels page
* Fill in Radius filter
* Click on "Find"
* See hotels in entered radius
